DLRMv2 changes (#58)
intel-ai-tce/ai-documents: AEM/mlperf/README.md

## Running Models with Intel Optimized Docker Image

### Set Directories
Set the directories on the host system where model, dataset, and log files will reside. These locations will retain model and data content between Docker sessions.
```
export DATA_DIR="${DATA_DIR:-${PWD}/data}"
export MODEL_DIR="${MODEL_DIR:-${PWD}/model}"
export LOG_DIR="${LOG_DIR:-${PWD}/logs}"
```

> ### [DLRMv2 Only] Download the Dataset [one-time operation]

Author: ```suggestion
### [Optional] Download the Dataset [one-time operation]
```

bash download_model.sh
```

> ### [DLRMv2 Excluded] Download the Dataset [one-time operation]

Author: ```suggestion
### Download the Dataset [one-time operation]
>**NOTE** Please bypass this step for DLRMv2 model.
```

Run this step inside the Docker container.  This is a one-time operation which will preserve the dataset on the host system using the volume mapping above.
```
bash download_dataset.sh
```

> ### [DLRMv2 Only] Prepare the Dataset [one-time operation]

Author: ```suggestion
### [Optional] Prepare the Dataset [one-time operation]
>**NOTE** This section only applies to DLRMv2 model.
```
